doc/source/fq_nmod_sparse_vec.rst

```diff
@@ -0,0 +1,176 @@
+.. _fq-nmod-sparse-vec:
+
+**fq_nmod_sparse_vec.h** -- sparse vectors over finite fields (word-size characteristic)
+===============================================================================
+
+Description.
+
+Types, macros and constants
+-------------------------------------------------------------------------------
+
+.. type:: fq_nmod_sparse_entry_t
+
+    Holds a pair (ind, val), where ind (of type slong) is an index into the
```

Comment:
The rst format has a way to specify types. The *'s in the types in the descriptions actually screw it up, so they should all be using the specialised format. Occurs in numerous functions below.